src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/CreateInvitation/ProjectRepositoryTestDouble.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;

```

Comment:
I don't understand what this is. I see it's used in 2 tests in CreateInvitationCommandHandlerTests ... why not use Mock?